js/scenes/Events/Events.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { View, Text } from 'react-native';
+
+import { styles } from './styles'
+
+const Events = ({ data }) => (
+    <View style={styles.container}>
```

Comment:
What is this data prop? It's confusing to have props which are unused. Please remove unused props.
Install ESLint to get warnings about unused variables.